lang/java/reef-tests/src/test/java/org/apache/reef/tests/TestEnvironment.java

```diff
@@ -55,4 +55,6 @@
   int getTestTimeout();
 
   LauncherStatus run(final Configuration driverConfiguration);
+
+  LauncherStatus run(final Configuration runtimeConfiguration, Configuration driverConfiguration);
```

Comment:
I need to set to the runtime the rack names.. In RackAwareEvaluatorTest.testRackAwareEvaluatorRunningOnRack1, 
I'm setting that value. That's is the reason. Maybe there's a better way to do this?